packages/node-runtime-worker-thread/src/serializer.ts

```diff
@@ -0,0 +1,145 @@
+import v8 from 'v8';
+import { ShellResult } from '@mongosh/shell-evaluator';
+import { inspect } from 'util';
+import { EJSON, Document } from 'bson';
```

Comment:
I noticed that some parts of `bson` library are re-exported from `service-provider-core`, but not EJSON that I'm using here. Not sure what was the reason for it, so I'm wondering should I just add EJSON as re-export also or is it okay to import from `bson` directly?

Reply:
It’s a good question, but I think it would be fine to load it here directly for our purposes